manifests/bucketeer/charts/web/values.yaml

```diff
@@ -1,24 +1,112 @@
-fullnameOverride: "web-gateway"
+image:
+  repository: ghcr.io/bucketeer-io/bucketeer-web
+  pullPolicy: IfNotPresent
+fullnameOverride: "web"
 namespace: default
 env:
+  cloudService:
+  profile: true
```

Comment:
We can delete this because we have it here.
https://github.com/bucketeer-io/bucketeer/pull/1181/files#diff-1bed54f9dc69221f401f6b8db27cdae14defa3f609a8d2e924f37de7f71752a9R13

```suggestion
```

Reply:
This line is default value for production environment, it should be `ture`.
For dev container, it should be `false`.